c/extras/aggregator.cc

```diff
@@ -0,0 +1,337 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// © H2O.ai 2018
+//------------------------------------------------------------------------------
+
+#include "datatable.h"
+#include "extras/aggregator.h"
+#include <stdlib.h>
+#include "utils/omp.h"
+#include "py_utils.h"
+#include "rowindex.h"
+#include "types.h"
+#include "datatable_check.h"
+
+Aggregator::Aggregator(DataTable* dt)
+  : dt_in(dt)
+{
+  create_dt_out();
+}
+
```

Comment:
where's `~Aggregator()` ?

Reply:
This `dt_out` is the dataframe created and returned by the aggregator to the user. I'm not sure we should destroy it when the aggregator is destroyed. Do you think we should?